### DISCUSSION TITLE
Debugging `Serializing big strings` warning in filesystem cache

### DISCUSSION THREAD

**Discussion by vsumner** [webpack/webpack#14936] [Q&A]
I'm looking for any tips/tricks on debugging the following warning when using the filesystem cache.
```
<w> [webpack.cache.PackFileCacheStrategy] Serializing big strings (282kiB) impacts deserialization performance (consider using Buffer instead and decode when needed)
```

**Comment by alexander-akait** [comment]
Can you provide `npx webpack-cli info`?

**Reply by vsumner** [reply]
```
  System:
    OS: macOS 12.0.1
    CPU: (12) x64 Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz
    Memory: 55.16 MB / 32.00 GB
  Binaries:
    Node: 16.10.0 - ~/.nvm/versions/node/v16.10.0/bin/node
    Yarn: 1.22.15 - ~/yarn/1.22.15/bin/yarn
    npm: 7.24.0 - ~/.nvm/versions/node/v16.10.0/bin/npm
  Browsers:
    Chrome: 96.0.4664.93
    Safari: 15.1
  Packages:
    copy-webpack-plugin: ^10.0.0 => 10.0.0
    css-loader: ^6.2.0 => 6.5.1
    html-webpack-plugin: ^5.5.0 => 5.5.0
    null-loader: ^4.0.1 => 4.0.1
    postcss-loader: ^4.2.0 => 4.2.0
    webpack: ^5.65.0 => 5.65.0
```

**Reply by alexander-akait** [reply]
do you have big HTML file (template for HTML)?

**Reply by vsumner** [reply]
No. I suspect it might be i18n translation files, but I'm hoping you might have suggestions that would allow me to track down where the problem might be.

**Reply by alexander-akait** [reply]
If you can provide configuration file maybe i have more ideas, anyway you can search this warning in webpack code (in `node_modules`) and debug it, it allows to understand what is module do it

**Reply by vsumner** [reply]
Thanks! I can't easily share yet but I'll see what I can learn from debugging.

**Comment by sokra** [comment]
Setting `infrastructureLogging.debug: /PackFileCache/` should print a stack trace for this warning
